sdk/communication/Azure.Communication.CallingServer/src/Models/CallingServerErrorResponse.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.Communication.CallingServer
+{
+    [CodeGenModel("CommunicationErrorResponse")]
+    public partial class CallingServerErrorResponse
```

Comment:
Does this need to be public?

Reply:
@navali-msft please take a look here.

Reply:
Removed CallingServerError as it is not used anywhere. Kept the one generated by sdk as internal.